### DISCUSSION TITLE
輸入框增強不能使用

### DISCUSSION THREAD

**Discussion by p23518** [immersive-translate/immersive-translate#718] [Help]
### 概述 | Summary

您好，一開始install時，輸入框增強可以使用在google及其他輸入框內，例如: //測試   ，就會自動轉換成test，但安裝一陣子後，突然不能用，請問有可能的問題是甚麼?

**Comment by theowenyoung** [comment]
试试看 github 的输入框能不能用？   或者看看设置里的输入框增强设置，看看有没有变化。
